src/ai/backend/manager/config.py

```diff
@@ -416,6 +416,7 @@
             ).allow_extra("*")
             | t.Null
         ),
+        t.Key("roundrobin_state", default=None): t.Null | t.String,
```

Comment:
In case of defining customized type, we can validate the input string here. Then we do not even need to handle `JSONDecodeError` when load json from this value.